src/Kdyby/Events/DI/EventsExtension.php

```diff
@@ -71,7 +71,8 @@ public function loadConfiguration()
 		$config = $this->getConfig($this->defaults);
 
 		$userConfig = $this->getConfig();
-		if (!isset($userConfig['debugger']) && !$config['debugger']) {
+		$isCli = in_array(php_sapi_name(), ['cli', 'phpdbg'], TRUE);
+		if ($isCli || (!isset($userConfig['debugger']) && !$config['debugger'])) {
```

Comment:
> For example I have an extension, that forcefully renders the the panel into a html file if it's not visible.

That sounds like a non-semantic hack though, debug panel should be limited to modes where tracy panel renders. Obviously I don't known enough what the tool you are referring to does, but I would be cleaner registering it's own listeners and not using panel as event storage.

> I would be more inclined to change the default value to off, so you have to explicitly enable the panel.

Yep, I considered that before and it would be sufficient fix. I still believe panel branches should not execute in cli mode though.